Python/frozen_modules/MANIFEST

```diff
@@ -0,0 +1,143 @@
+# This file is auto-generated by Tools/scripts/freeze_modules.py.
```

Comment:
Specifically, it would cause the "check_generated_files" job on GitHub CI to fail if the PR's branch was pushed before "make regen-frozen" was run (and the .py files changed, etc.).  In general the manifest offers an easier-to-read snapshot of the frozen modules, as opposed to freeze_modules.py, frozen.c, etc.

Regardless, I added a comment at the top with a little more info.  Also, for now I've added the frozen .h files back into the repo, so MANIFEST doesn't fill the "check_generated_files" role at the moment.